### DISCUSSION TITLE
Apache Configuration Help

### DISCUSSION THREAD

**Discussion by o62** [pretix/pretix#2008] [Q&A]
Hi folks. I'm trying to get a manual installation created following [this guide](https://docs.pretix.eu/en/latest/admin/installation/manual_smallscale.html). Everything was cake except for the configuration for the reverse proxy in apache. I took a stab on it but when I go to tickets.mydomain.com I'm getting an internal server error. Translating the Nginix to Apache is what gave me trouble. Any help would be great:
```
 <VirtualHost *:80>
        ServerAdmin admin@mydomain.com
	ServerName tickets.mydomain.com
	ServerAlias www.tickets.mydomain.com
        DocumentRoot /var/pretix/data/media/
	ProxyPass / http://127.0.0.1:8345/
   	ProxyPassReverse / http://127.0.0.1:8345/
    RequestHeader set X-Forwarded-Port "443"
    RequestHeader set X-Forwarded-Proto "https"

        <Directory /var/pretix/data/media/>
             AllowOverride All
            Require all granted
        </Directory>

        <Directory /var/pretix/venv/lib/python3.6/site-packages/pretix/static.dist/>
             AllowOverride All
            Require all granted
        </Directory>


Alias /media "/var/pretix/data/media/"



Alias /static "/var/pretix/venv/lib/python3.6/site-packages/pretix/static.dist/"



<Directory /var/pretix/data/media/cachedfiles>
order deny,allow
deny from all
</Directory>

<Directory /var/pretix/data/media/invoices>
order deny,allow
deny from all
</Directory>



        ErrorLog ${APACHE_LOG_DIR}/error.log
        CustomLog ${APACHE_LOG_DIR}/access.log combined

        <IfModule mod_dir.c>
            DirectoryIndex index.php index.pl index.cgi index.html index.xhtml index.htm
        </IfModule>
RewriteEngine on
RewriteCond %{SERVER_NAME} =tickets.mydomain.com [OR]
RewriteCond %{SERVER_NAME} =www.tickets.mydomain.com
RewriteRule ^ https://%{SERVER_NAME}%{REQUEST_URI} [END,NE,R=permanent]
</VirtualHost>
```

**Comment by raphaelm** [comment]
An Internal Server Error is probably unrelated to your Apache configuration. Can you have a look into pretix.log?

**Reply by o62** [reply]
It is empty, which is why I think it is an apache error. In the apache log, the only thing I am seeing is this: AH01144: No protocol handler was valid for the URL / (scheme 'http'). If you are using a DSO version of mod_proxy, make sure the proxy submodules are included in the configuration using LoadModule.

**Reply by raphaelm** [reply]
An internal server error in pretix design is not caused by apache for sure. There needs to be some log entry by pretix somewhere.

**Comment by sysadminpower2019** [comment]
any updates on this? I really would like to have apache support @raphaelm

**Reply by raphaelm** [reply]
We will not start adding more webservers to the docs, but you can find examples to start from in the issues:
https://github.com/search?q=repo%3Apretix%2Fpretix+apache&type=issues

e.g. https://github.com/pretix/pretix/issues/1237

**Comment by sysadminpower2019** [comment]
@raphaelm unfortunately that thread does not provide any information on how to resolve the CSRF cookie issue. I wish this was not set as the default as it complicates testing use cases. I have how exactly to solve this CSRF issue with apache.
